src/Clause.ts

```diff
@@ -180,6 +196,16 @@ export default class Clause extends Builder {
     }
   }
 
+  isEffectApplicable(effectName: string) {
+    // The following effects are runtime only:
+    //
+    // uc: Only runtime can call user code.
+    if (this.type === 'static sdo') {
```

Comment:
Nothing currently has this type, and the logic [elsewhere](https://github.com/tc39/ecmarkup/blob/a654534daa79e5af43987203f81bfa1c6e726831/src/header-parser.ts#L325-L360) doesn't support it. If the intention is just to exclude "Static Semantics" SDOs, we could track the "Static Semantics" part as an attribute separately from the type.

Reply:
Ah that's right, I was testing off a local copy of ecma262 where I had annotated static SDOs. I'll make a PR to ecma262.

Reply:
Switched to using "Static Semantics" prefix.